jenkins_jobs/modules/hipchat_notif.py

```diff
@@ -131,12 +133,26 @@ def gen_xml(self, xml_parent, data):
                 logger.warning("'room' is deprecated, please use 'rooms'")
                 hipchat['rooms'] = [hipchat['room']]
 
-        properties = xml_parent.find('properties')
-        if properties is None:
-            properties = XML.SubElement(xml_parent, 'properties')
-        pdefhip = XML.SubElement(properties,
-                                 'jenkins.plugins.hipchat.'
-                                 'HipChatNotifier_-HipChatJobProperty')
+        plugin_info = self.registry.get_plugin_info("Jenkins HipChat Plugin")
+        version = pkg_resources.parse_version(plugin_info.get('version', '0'))
+
+        if version >= pkg_resources.parse_version("0.1.9"):
+            publishers = xml_parent.find('publishers')
+            if publishers is None:
+                publishers = XML.SubElement(xml_parent, 'publishers')
+
+            logger.warning(
+                "'hipchat' module supports the old plugin versions <1.9, "
+                "newer versions are supported via the 'publishers' module. "
+                "Please upgrade you job definition")
+            return hipchat_pub(self.registry, publishers, hipchat)
```

Comment:
@hrubi Basically, the "dispatch" will call "publisher" module, then executes the hipchat function (which is imported as hipchat_pub in this PR). I have not found the root cause why "dispatch" can not work yet. It may related to bunch of other files in master... So, can we go with this for now, other relating things can be debug when we want to merge master to our fork?